### ISSUE TITLE
http status is ignored after upgrading to 1.0.7 from 1.0.5

### ISSUE THREAD

**Issue by thynson** [Yegorich555/webpack-mock-server#16]
```javascript
export default webpackMockServer.add((app, helper) => {
  app.use(bodyParser.urlencoded());
  app.use(bodyParser.json());

  app.post('/test', (req, res) => {
    res.status(403).json({});
  });
})
```
and then `curl localhost:8080/test  -X POST` 
```
*   Trying ::1:8080...
* Connected to localhost (::1) port 8080 (#0)
> POST /test HTTP/1.1
> Host: localhost:8080
> User-Agent: curl/7.76.1
> Accept: */*
> 
* Mark bundle as not supporting multiuse
< HTTP/1.1 201 Created
< x-powered-by: Express
< content-type: application/json; charset=utf-8
< content-length: 2
< etag: W/"2-vyGp6PvFo4RvsFtPoIWeCReyIC8"
< date: Wed, 23 Jun 2021 15:00:19 GMT
< connection: close
< 
* Closing connection 0
{}
```

**Comment by thynson** [comment]
Just tested v1.0.6 that it is also broken.

**Comment by thynson** [comment]
Shall be caused by: https://github.com/Yegorich555/webpack-mock-server/commit/330680b31c7adef5832dd70254976cbb3d0c2177#diff-7b859596227dab84e5bcc89df52775017ac775fab9a7b96c445c093b77963b3cR44

**Comment by Yegorich555** [comment]
Thanks. Fixed with [v1.0.8](https://github.com/Yegorich555/webpack-mock-server/compare/v1.0.7...v1.0.8)
